README.md

```diff
@@ -25,12 +25,21 @@ def uplink_callback(msg, client):
   print("Received uplink from ", msg.dev_id)
   print(msg)
 
+handler = ttn.HandlerClient(app_id, access_key)
 
-my_client = mqtt_client(app_id, access_key)
-my_client.set_uplink_callback(uplink_callback)
-my_client.connect()
+# using mqtt client
+mqtt_client = handler.data()
+mqtt_client.set_uplink_callback(uplink_callback)
+mqtt_client.connect()
 time.sleep(60)
-my_client.close()
+mqtt_client.close()
+
+# using application manager client
+app_client =  handler.application()
+my_app = app_client.get()
```

Comment:
Does `app_client.get()` return a single app or does it return multiple apps?

Reply:
this returns a single app, using the get method of the ApplicationClient class. This is explained in the documentation of ApplicationClient class already. Should I precise it in this example?

Reply:
It's good :)